schematic/schemas/data_model_parser.py

```diff
@@ -0,0 +1,319 @@
+        label_jsonld_key, subclassof_jsonld_key, id_jsonld_key, dn_jsonld_key = [self.rel_dict[key]['jsonld_key']
+                                                    for key in jsonld_keys_to_extract ]
+        
+        # Gather all labels from the model.
+        model_labels = [v[label_jsonld_key] for v in model_jsonld]
+
+        attr_rel_dictionary = {}
+        # Move through each entry in the jsonld model
+        for entry in model_jsonld:
+            # Get the label of the entry
+            try:
+                # Get the entry display name (if recorded)
+                entry_name = entry[dn_jsonld_key]
+            except:
+                # If no display name, get the label.
+                entry_name = entry[label_jsonld_key]
+
+            # If the entry is an attribute that has not already been added to the dictionary, add it.
+            if entry_name not in attr_rel_dictionary.keys():
+                attr_rel_dictionary.update({entry_name: {'Relationships': {}}})
+            
+            # Add relationships for each attribute
+            # 
+            # Go through each defined relationship type (key) and its attributes (val)
+            for key, val in self.rel_dict.items():
+                # Determine if current entry can be defined by the current reationship.
+                if val['jsonld_key'] in entry.keys() and 'csv_header' in val.keys():
+                    # Retrieve entry value associated with the given relationship
+                    rel_entry = entry[val['jsonld_key']]
+                    # if there is an entry treat it by type and add to the attr:relationships dictionary.
+                    if rel_entry:
+                        # Retrieve ID from dictionary single value dictionary
+                        if type(rel_entry) == dict and len(rel_entry.keys()) == 1:
+                            parsed_rel_entry = entry.get(val['jsonld_key'])['@id']
+                        # Parse list of dictionaries to make a list of entries with context stripped (will update this section when contexts added.)
+                        elif type(rel_entry)==list and type(rel_entry[0]) == dict:
+                            parsed_rel_entry = [r[id_jsonld_key].split(':')[1] for r in rel_entry]
+                        # Strip context from string and convert true/false to bool
+                        elif type(rel_entry) == str:
+                            # Remove contexts and treat strings as appropriate.
+                            if ':' in rel_entry and 'http:' not in rel_entry:
+                                parsed_rel_entry = rel_entry.split(':')[1]
+                                # Convert true/false strings to boolean
+                                if parsed_rel_entry.lower() =='true':
+                                    parsed_rel_entry = True
+                                elif parsed_rel_entry.lower == 'false':
+                                    parsed_rel_entry == False
```

Comment:
Should this be `parsed_rel_entry = False`?

Reply:
Thank you fo catching this!